eisenhower-matrix/src/main/java/com/google/sps/servlets/GetTask.java

```diff
@@ -158,6 +150,15 @@ private void deleteTask(Long id) {
             tasks.removeIf(obj -> obj.getId() == id);
             datastore.delete(taskEntityKey);
     }
+
+    private Task entityToTask(Entity e) {
+        String name = (String)e.getProperty("name");
+        Date date = (Date)e.getProperty("date");
+        Long importance = (Long)e.getProperty("importance");
```

Comment:
The importance field should be an `int/enum/string` right? Why are you using a `long`?

Reply:
Datastore stores all integer values as `Long`. When I create the Task I convert them into `int` values, but if I try to directly store them to `int` I get a compilation error

Reply:
@deeptivaidy @EduardoSantos7 you have to use long; pls learn from my trials and errors :(